tests/integration/test_basic.py

```diff
@@ -195,19 +218,12 @@ def _make_error_result(download_error, source="microsoft", bucket_type="http"):
                 "source": source,
             },
         ]
-        if bucket_type != "http":
```

Comment:
removed this because i fixed an error where i'd incorrectly left out the layout field on the non-http sources in the permissions tests, causing them to include dSYMs in their output.